docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,382 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
+         - `*.coredump`: A process memory dump.
+         - `*.so_info`: Contains information about shared libraries used by the crashed application, together with the names of packages that contain them.
+         - `*.log`: Contains information provided by `dump_systemstate`.
+
+     Default value: `FULL`
+
+
+-  **ExtraScript**  
+     Path to a script or an application that will be executed during report creation. The process will be started asynchronously.
+
+     Default value is empty
+
+#### Disabling crash-manager
+
+In some situations there may be a need to disable the crash-manager functionality. In such cases, the recommended solution is provide the sysctl configuration file to overwrite the default `kernel.core_pattern` setting. Example file `99-disable-crash-manager.conf` should be placed in `/usr/lib/sysctl.d/` directory and should contain:
+
+```
+kernel.core_pattern=
+kernel.core_uses_pid=0
+```
+
+### D-Bus notification
+
+After the report is generated, crash-manager sends the D-Bus signal with information about the crashed process.
+
+Signature: `sssssiia{sv}`  
+Member: `ProcessCrashed`  
+Interface: `org.tizen.system.crash.Crash`  
+Path: `/Org/Tizen/System/Crash/Crash`  
+Data structure:  
+
+Data         | Type
+-------------|-------
+command name | STRING
+command path | STRING
+appid        | STRING
+pkgid        | STRING
+report path  | STRING
+PID          | INT32
+TID          | INT32
+array        | ARRAY
+
+Array containing pairs of keys (STRING) and values (VARIANT). Possible keys:
+
+Key          | Type   | Description
+-------------|--------|---------------
+x86.eip      | UINT32 | instruction pointer
+x86_64.rip   | UINT64 | instruction pointer
+arm.pc       | UINT32 | instruction pointer
+arm.lr       | UINT32 | instruction pointer
+aarch64.pc   | UINT64 | instruction pointer
+aarch64.lr   | UINT64 | instruction pointer
+sys.signal   | INT32  | signal that caused crash
+sys.tid.comm | STRING | thread name
+
+Depending on the architecture, array will contain:
+
+Arch    | PC (instruction pointer) | Link Register
+--------|--------------------------|------------------
+x86     | x86.eip                  | (not available)
+x86_64  | x86_64.eip               | (not available)
+arm     | arm.pc                   | arm.lr
+aarch64 | aarch64.pc               | aarch64.lr
+
+### minicoredumper
+
+minicoredumper is a tool that accepts a core dump on a standard input and saves the modified (size-reduced) core dump in a file. This tool is internally used by crash-manager to generate minicoredump. Modifications depends on the configuration, and they mainly rely on minimization of the core dump file.
+
+Usually, integrator does not need to modify this configuration unless integrator needs to change:
+
+-   parameters of minicoredump files
+-   for which applications minicoredump is created and for which regular core dump is created
```

Comment:
Please as a period at the end of the sentence as it is a continuation of the leading sentence.